arch/arm/boot/dts/overlays/piscreen-overlay.dts

```diff
@@ -98,5 +100,7 @@
 		fps =		<&piscreen>,"fps:0";
 		debug =		<&piscreen>,"debug:0";
 		xohms =		<&piscreen_ts>,"ti,x-plate-ohms;0";
+		drm =		<&piscreen>,"compatible=waveshare,rpi-lcd-35",
+				<&piscreen>,"reset-gpios:8=GPIO_ACTIVE_HIGH";
```

Comment:
Are you sure this works?:
$ dtmerge arch/arm/boot/dts/bcm2711-rpi-4-b.dtb base.dtb -
$ dtmerge base.dtb merged.dtb arch/arm/boot/dts/overlays/piscreen.dtbo drm
DTOVERLAY[error]: invalid override value 'GPIO_ACTIVE_HIGH' - ignored
```

Reply:
You're right - my update script to push the change to the Pi obviously didn't for some reason (probably PEBKAC).